server/metrics/metrics.go

```diff
@@ -563,6 +563,14 @@ var (
 		Help:      "Age of the last entry evicted from the executor's local file cache (relative to when it was added to the cache), in **microseconds**.",
 	})
 
+	FileCacheFileSizeBytes = promauto.NewHistogram(prometheus.HistogramOpts{
+		Namespace: bbNamespace,
+		Subsystem: "remote_execution",
+		Name:      "file_cache_file_size_bytes",
```

Comment:
Should we call the metric & variable file_cache_added_file_size_bytes to be more clear that this represents files being added?

Reply:
Good call -- done